webrender/src/device/gfx/device.rs

```diff
@@ -1070,9 +1070,9 @@ impl<B: hal::Backend> Device<B> {
             frame_images,
             viewport,
             if present_mode == hal::window::PresentMode::Mailbox {
-                (caps.image_count.end - 1).min(3) as usize
+                (caps.image_count.end() - 1).min(3) as usize
```

Comment:
since `image_count` is inclusive on the right side, maybe we don't need "-1" ?

Reply:
Fixed in https://github.com/szeged/webrender/pull/306/commits/16cf3a4d9b636611ce8f4a099ed23a4bfd68f0f0